### ISSUE TITLE
Update Sketch wireframes

### ISSUE THREAD

**Issue by joshsmith** [code-corps/design#7]
We should add symbols and styles to our wireframes so we can more easily work from a shared design library.

Styles
- [ ] Add text styles for all combinations of size, weight, and color used in the app

Symbols
- [ ] Inputs
- [ ] Buttons
- [ ] Site header
- [ ] Site footer
- [ ] Project card
- [ ] Task card

Screens
- [ ] Index
- [ ] About
- [ ] Login
- [ ] Organization
- [ ] Organization profile
- [ ] Privacy
- [ ] Project
- [ ] Project checkout
- [ ] Project donate
- [ ] Project contributors
- [ ] Project donation payment settings
- [ ] Project donation goal settings
- [ ] Project profile
- [ ] Project tasks
- [ ] Project tasks new
- [ ] Project task
- [ ] Project thank you
- [ ] Projects
- [ ] Organization projects
- [ ] Settings profile
- [ ] Signup
- [ ] Password forgot
- [ ] Password reset

**Comment by pigstake** [comment]
@JoshSmith Happy to take a look at this. Is there a Sketch file now? @daveconnis mentioned a Dropbox over Slack.

**Comment by joshsmith** [comment]
@pigstake sorry, missed this notification (way down in the queue here!). There is a Sketch file! Maybe we should schedule a time to chat and look through everything we've got, and I know @daveconnis has been wanting that to happen for some time now. ⌚️  😄

**Comment by pigstake** [comment]
@joshsmith Yeah sounds good! Next few weeks are increasingly jammed for me as I prep to move and go nomadic for the next few months. I'll slack you about times.
